formfactory/models.py

```diff
@@ -291,6 +304,7 @@ class FormField(models.Model):
     additional_validators = models.CharField(
         choices=ADDITIONAL_VALIDATORS, max_length=128, blank=True, null=True
     )
+    additional_validators = models.ForeignKey(Validator, blank=True, null=True)
```

Comment:
@qoda it was an easy way to get to use the `as_function` method the same way its used in `Action`.
Now that I think about it, I don't really need this to be a FK. I could leave as is and lookup the validator directly from `_registry` before using it. Thoughts?